### PR TITLE
Feature/eu applicant form

### LINKED ISSUE #43: Add functionality to POST an Applicant
- [x] Build component (ApplicantForm)
- [x] Upon load, form is displayed to fill out details
- [x] Create button (calls fetch function `postNewApplicant`)
`POST /api/v1/applicants/`
- [x] If successful, will route to `applicant/:id`
- [x] route to applicantForm from `I'm an applicant!`
- [x] prevent double fetch on create new applicant
- [x] Nav links for my Profile and Inbox appear upon "Auth"

### LINKED ISSUE #47: ApplicantForm style
- [x] make sure columns are lined up at top
- [x] text input smaller
- [x] bio three times height
- [x] align all text left
- [x] accent text color
- [x] add submit button
- [ ] refresh button toggle on click
- [x] add bio before first name w message

### LINKED ISSUE #48: Add functionality to create a new applicant
- [x] A user should be able to fill out a form and select values to create an anonymous and unique profile.
- [x] Inputs need to be captured and converted to proper data.
- [x] Upon submit, that users profile should post and save to the data base.

### PR DESCRIPTION
**Pull request by ErinUntermeyer**
#### Type of Change Made 
- [x] bugfix 
- [x] new feature 
- [ ] breaking change 
- [ ] testing 
- [ ] formatting 
- [x] styling
#### What's this PR do?
- Updates search icon, conditionally renders based on open or closed
- Changes inputs to textareas in the ApplicantForm component
- Fixes a bug with the clear button to set state to empty arrays
- Adds functionality to GET the possible skills and values options from database upon ApplicantForm load
- Adds functionality to POST a new applicant to the database
- Adds redirect to go to `applicant/:id` upon a successful post
#### Where should the reviewer start?
ApplicantForm.tsx
#### How should this be manually tested?
Go to `http://localhost:3000/create-applicant` and make a new user!
#### Any background context you want to provide?
- There's a zero at the bottom of the form rendering, needs to be removed
- I think the next steps should be hiding the nav links to "My Profile" and "Inbox" until the App is in "Applicant mode". Still need to figure out how we will set that. Will it just be upon clicking "I'm an Applicant"? If so, how do you change back to the Employer?
#### What are the relevant tickets?
- Closes #43 closes #48 closes #47 
#### Screenshots (if appropriate)
Screen Shot 2020-10-30 at 6 53 34 PM

### CODE REVIEW

File: src/components/Applicant/Applicant.tsx [hunk 1]
```diff
@@ -72,11 +72,11 @@ const determineMatchedAttribute = (attribute: string, keyword: string, props: an
 					</div>
 				</div>
 				<div className="applicant-attributes">
-					<div className="attribute-box">
+					<div>
 						<h5>Skills</h5>
 						{attributeLists.skillTags}
 					</div>
-					<div className="attribute-box">
+					<div>
```
Comment (GreysonElkins) [inline src/components/Applicant/Applicant.tsx:79]: I'm glad we caught this class name overlap!

File: src/components/ApplicantForm/ApplicantForm.tsx [hunk 2]
```diff
@@ -105,40 +132,70 @@ const makeTags = (tags:Array<{attribute:string, id:number}>) => {
...
-				<button className="cta-button">create your profile</button>
+			<button
+				className="cta-button"
+				style={{ margin: '1em auto' }}
+				onClick={createNewApplicant}
+			>
+				create your profile
+			</button>
+			{successfulPost &&
+				<Redirect
```
Comment (GreysonElkins) [inline src/components/ApplicantForm/ApplicantForm.tsx:195]: A great start! We may want to come back and review this; this way requires an additional fetch on the next page which we can prevent by passing props from this component. I think this is great for starting out though and keeping our workflow tight.

File: src/components/Header/Header.test.tsx [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+describe('Header Component', () => {
+
+	it('Should display a title', () => {
+		render(<MemoryRouter><Header /></MemoryRouter>)
+		const title = screen.getByRole('heading', { name: /hire up/i })
+		expect(title).toBeInTheDocument()
+	})
+
+	it('Should display a search icon', () => {
```
Comment (GreysonElkins) [inline src/components/Header/Header.test.tsx:14]: Thanks for starting on these!

**Review by GreysonElkins — APPROVED** [review]
🙌

File: src/components/ApplicantForm/ApplicantForm.tsx (hunk 2 above)
Comment (ErinUntermeyer) [reply src/components/ApplicantForm/ApplicantForm.tsx:195]: Agreed. We should be passing the data returned from the successful POST into Applicant and using that to display. For readability, I'd prefer to have successfulPost be a boolean and applicant or newApplicant or something that holds the response.

File: src/components/Search/Search.test.tsx [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+	// 	// find one of them (getByText???)
+	// 	// click it
+	// 	// see that state updated with it?
+	// 	// when clear button clicked, see that state is empty?
+	// 	const option1 = screen.getByRole('button', { name: /buttonname/i})
+	// 	fireEvent.click(option1)
+	// 	fireEvent.click(clearButton)
+	// 	expect(clearButton).toBeInTheDocument()
+	// })
+
```
Comment (amykarnaze) [inline src/components/Search/Search.test.tsx:50]: Nice fireEvent tests!

**Review by amykarnaze — APPROVED** [review]
Great job implementing a tricky useReducer and getting a successful post going.

**Pull request merged**